src/components/Image/BackgroundImage.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useRef } from "react";
+import observerOptions from "./options";
+import Wrapper from "./Wrapper";
+import { ImageProps } from "./types";
+
+const BackgroundImage: React.FC<ImageProps> = ({ src, ...otherProps }) => {
+  const imgRef = useRef(null);
+
+  useEffect(() => {
+    const img = (imgRef.current as unknown) as HTMLElement;
```

Comment:
You can type the ref on line 7.

```
const imgRef = useRef<HTMLElement | null>(null);
```

Reply:
it needs to be casted as a non null element for use line 20